### ISSUE TITLE
Accessibility test for jupyterLite

### ISSUE THREAD

**Issue by smythp** [DHRI-Curriculum/DHRIFT#72]
@szweibel  I'll put any findings in this issue just to surface the work).

**Comment by smythp** [comment]
OK, so I tested this. Here are my notes:

In the JupyterLab version (there is also a retrolab version not tested), the menu for opening notebooks, console, etc. is broadly accessible with minor issues related to labeling

Once you access a notebook, there are some issues:

- Moving among cells doesn't have screen reader feedback, it says things like "blank" or gives no output
- It should have more and better alerts when running cells, but you do hear the output of the cell when you run it
- It seems that the toolbar works and is labeled, which is cool
- Landmarks have been implemented so you have move between toolbar, the main notebook view, and the files, which is also good

I only tested basic functionality. My initial asssessment is that if a few things were changed, the project would meet some minimum threshold for use with screen readers.
